src/v/cloud_storage_clients/configuration.cc

```diff
@@ -92,10 +92,13 @@ ss::future<s3_configuration> s3_configuration::make_configuration(
     client_cfg.secret_key = skey;
     client_cfg.region = region;
     client_cfg.uri = access_point_uri(endpoint_uri);
-    client_cfg.url_style = overrides.url_style;
```

Comment:
Is overrides.url_style important to consider anymore? Should we remove it from the overrides?

Reply:
Oops, no, it has no uses past this point. Will remove.